### PR TITLE
Fixes #3213 Add byte stream example

### LINKED ISSUE #3213: Get-Content topic: explain the relationship between -Raw and -AsByteStream / -Encoding Byte
Combining `-Raw` with `-AsByteStream` / `-Encoding Byte` is actually the simplest way to get the file's content as a raw byte array (`[byte[]]`), output as a _single_ object (as opposed to the bytes streaming one by one, in the absence of `-Raw`).

The description of both parameters should reflect this option.

Note that the currently documented method for getting all bytes as a single array is to combine  <br>`-AsByteStream` / `-Encoding Byte`   with `-ReadCount 0` instead, which works too, but:

* using `-Raw` instead is more obvious and easier to type (assuming you know that [`-Raw` actually means: read the file _as a whole_](https://github.com/PowerShell/PowerShell/issues/7715))
* `-ReadCount 0` outputs an `[object[]]` array rather than a specifically typed `[byte[]]` array (though the difference will rarely matter).

Version(s) of document impacted
------------------------------
- [x] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

### PR DESCRIPTION
**Pull request by bobbytreed**
Version(s) of document impacted
------------------------------
- [ ] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[893d404](https://github.com/bobbytreed/PowerShell-Docs/pull/4374/commits/893d4042ee11594c053506ae94a86c7e4e6d6c58)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 6 sec(s)_, this is based on the last _72_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[893d404](https://github.com/bobbytreed/PowerShell-Docs/pull/4374/commits/893d4042ee11594c053506ae94a86c7e4e6d6c58)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/3.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-3.0&branch=pr-en-us-4374) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/4.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-4.0&branch=pr-en-us-4374) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/5.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.0&branch=pr-en-us-4374) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/5.1/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.1&branch=pr-en-us-4374) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-6&branch=pr-en-us-4374) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C31%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905311548372589-4374%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=4GItm8GW8SZHr%2FStvxqYJBfKi6KdZpaPo0QnAfXuxVs%3D&st=2019-05-31T15%3A55%3A48Z&se=2019-07-01T16%3A00%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Get-Content.md [hunk 1]
```diff
@@ -222,6 +222,41 @@ Raw contains 1 lines.
...
+
+Name           MemberType            Definition
+----           ----------            ----------
+Count          AliasProperty         Count = Length
+Add            Method                int IList.Add(System.Object value)
+```
+
+The first command uses the **Encoding** parameter to get the stream of bytes from the file.
+The **Raw** parameter ensures that the bytes are returned as a `[System.Byte[]]`. If the **Raw**
+parameter was absent, the return value would be a stream of bytes, which would be interpreted by
```
Comment (sdwheeler) [inline reference/3.0/Microsoft.PowerShell.Management/Get-Content.md]: Passive voice: "would be"
Change to "is"

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[fb5b8a4](https://github.com/bobbytreed/PowerShell-Docs/pull/4374/commits/fb5b8a474c14f4bb35ac6c45d12bf096082ade20)_: 

:clock10: **Full build**: the average full build time is _13 min(s) 2 sec(s)_, this is based on the last _78_ build(s) for this repository.

**Comment by zjalexander** [comment]
Docs Build status updates of commit _[fb5b8a4](https://github.com/bobbytreed/PowerShell-Docs/pull/4374/commits/fb5b8a474c14f4bb35ac6c45d12bf096082ade20)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/3.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-3.0&branch=pr-en-us-4374) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/4.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-4.0&branch=pr-en-us-4374) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/5.0/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.0&branch=pr-en-us-4374) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/5.1/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-5.1&branch=pr-en-us-4374) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Get-Content.md](https://github.com/bobbytreed/PowerShell-Docs/blob/bobbytreed-issue3213/reference/6/Microsoft.PowerShell.Management/Get-Content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/get-content?view=powershell-6&branch=pr-en-us-4374) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C31%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201905311804320141-4374%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=VW%2BaCioGto1lLcKMGOjaXPge8IPySIoI4Ex4%2FrhzWKY%3D&st=2019-05-31T18%3A10%3A24Z&se=2019-07-01T18%3A15%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**